modules/manage/partials/schema-compatibility.adoc

```diff
@@ -42,4 +42,24 @@ Remove fields
 | Remove required fields 
 
 Change data types of fields
+
+| **JSON**
+a| Add optional properties
+
+Relax constraints, for example:
+
+* Decrease a `minimum` value or increase a `maximum` value
+* Decrease `minItems`, `minLength`, or `minProperties`; increase `maxItems`, `maxLength`, `maxProperties`
+* Add more property types (for example, "type": "integer" to "type": ["integer", "string"])
+* Add more enum values
+
+| Remove properties
+
+Add required properties
+
+Change property names and types
+
+Tighten or add constraints
```

Comment:
Would it be worth enumerating here just like with "Relax constraints" under the "Backward-compatible tasks" column?